website/client/js/controllers/tasksCtrl.js

```diff
@@ -100,8 +99,17 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
       }
     };
 
+    $scope.editTask = function(task, user) {
+      originalTasks.push(_.cloneDeep(task));
```

Comment:
This seems like a weird way to deal with this. I'd model it after how groups are edited.